demo/admin/src/products/future/ProductsGrid.cometGen.tsx

```diff
@@ -25,46 +28,60 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
     },
     columns: [
         {
-            type: "combination",
+            type: "virtual",
             name: "overview",
+            loadFields: ["category.title"],
             headerName: "Overview",
             minWidth: 200,
-            primaryText: "title",
-            secondaryText: {
-                // TODO: Change this to use the "group" type instead of "formattedMessage", once implemented (SVK-368)
-                type: "formattedMessage",
-                message: "{price} • {type} • {category} • {inStock}",
-                valueFields: {
-                    price: {
-                        type: "number",
-                        field: "price",
-                        currency: "EUR",
-                        emptyValue: "No price",
-                    },
-                    type: {
-                        type: "staticSelect",
-                        field: "type",
-                        values: typeValues,
-                        emptyValue: "No type",
-                    },
-                    category: {
-                        type: "text",
-                        field: "category.title",
-                        emptyValue: "No category",
-                    },
-                    inStock: {
-                        type: "staticSelect",
-                        field: "inStock",
-                        values: [
-                            { value: true, label: "In stock" },
-                            { value: false, label: "Out of stock" },
-                        ],
-                    },
-                },
+            renderCell: ({ row }) => {
+                const typeLabels: Record<string, ReactNode> = {
+                    Cap: <FormattedMessage id="product.overview.secondaryText.type.Cap" defaultMessage="great Cap" />,
+                    Shirt: <FormattedMessage id="product.overview.secondaryText.type.Shirt" defaultMessage="Shirt" />,
+                    Tie: <FormattedMessage id="product.overview.secondaryText.type.Tie" defaultMessage="Tie" />,
```

Comment:
Unimportant detail: message IDs should generally start lower-case

```suggestion
                    Cap: <FormattedMessage id="product.overview.secondaryText.type.cap" defaultMessage="great Cap" />,
                    Shirt: <FormattedMessage id="product.overview.secondaryText.type.shirt" defaultMessage="Shirt" />,
                    Tie: <FormattedMessage id="product.overview.secondaryText.type.tie" defaultMessage="Tie" />,
```